graylog2-web-interface/src/views/components/WidgetQueryControls.jsx

```diff
@@ -11,29 +10,22 @@ import DocumentationLink from 'components/support/DocumentationLink';
 import DocsHelper from 'util/DocsHelper';
 import Button from 'components/graylog/Button';
 import TopRow from 'views/components/searchbar/TopRow';
-import Widget from 'views/logic/widgets/Widget';
 import { StreamsStore } from 'views/stores/StreamsStore';
 import { SearchConfigStore } from 'views/stores/SearchConfigStore';
-import { WidgetActions } from 'views/stores/WidgetStore';
 import { GlobalOverrideActions, GlobalOverrideStore } from 'views/stores/GlobalOverrideStore';
 import GlobalOverride from 'views/logic/search/GlobalOverride';
 import SearchActions from 'views/actions/SearchActions';
-import { createElasticsearchQueryString } from 'views/logic/queries/Query';
 
 import TimeRangeTypeSelector from './searchbar/TimeRangeTypeSelector';
 import TimeRangeInput from './searchbar/TimeRangeInput';
 import StreamsFilter from './searchbar/StreamsFilter';
 import SearchButton from './searchbar/SearchButton';
 import QueryInput from './searchbar/AsyncQueryInput';
-import SearchBarForm from './searchbar/SearchBarForm';
-
-import { DEFAULT_TIMERANGE } from '../Constants';
 
 type Props = {
   availableStreams: Array<any>,
   config: any,
   globalOverride: ?GlobalOverride,
-  widget: Widget,
```

Comment:
The widget prop can also be removed in `WidgetQueryControls.test.jsx`

Reply:
:white_check_mark: